source/val/validate_memory.cpp

```diff
@@ -2135,6 +2140,9 @@ spv_result_t ValidateCooperativeMatrixLoadStoreKHR(ValidationState_t& _,
              << "Stride operand <id> " << _.getIdName(stride_id)
              << " must be a scalar integer type.";
     }
+  } else if (stride_required) {
```

Comment:
This could be further enhanced for stores by checking that stride > 0.

Reply:
Agree but I'd rather do it as a follow up change once https://github.com/KhronosGroup/SPIRV-Registry/issues/279 is resolved.